Documents/Help/Changelog.md

```diff
@@ -3,7 +3,7 @@
 =======
 
 ## 2024-06-24 - Build 2406 - June 2024
-* Resolved [#1528](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1528), Tracking colours need reviewing
+* Resolved [#1302](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1302), **[Breaking Change]** Font being used by "Professional" theme is pants !* Resolved [#1528](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1528), Tracking colours need reviewing
```

Comment:
Might need a new line here :)

Reply:
No, it's just the way the "Check diff" has done the drawing:
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/5d58c7d5-26e2-427e-97c1-e6569fa6079b)

Reply:
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/232c6e9b-99ed-4815-935c-4052ae317530)